src/System.Runtime/tests/System/String.cs

```diff
@@ -751,10 +753,15 @@ public static void TestIndexOf_HungarianDoubleCompression()
         string target = "ddzs";
         WithCulture(new CultureInfo("hu-HU"), () =>
         {
-            // TODO: [ActiveIssue(3972)]
-
-            //Assert.Equal(0, source.IndexOf(target));
-            //Assert.Equal(0, source.IndexOf(target, StringComparison.CurrentCulture));
+            /* 
+             There are differences between Windows and ICU regarding contractions.
+             Windows has equal contraction collation weights, including case (target="Ddzs" same behavior as "ddzs").
+             ICU has different contraction collation weights, depending on locale collation rules.
+             If CurrentCultureIgnoreCase is specified, ICU will use 'secondary' collation rules
+              which ignore the contraction collation weights (defined as 'tertiary' rules)
+            */
+            Assert.Equal(s_isWindows ? 0 : -1, source.IndexOf(target));
+            Assert.Equal(s_isWindows ? 0 : -1, source.IndexOf(target, StringComparison.CurrentCulture));
```

Comment:
Is there a corresponding test for non-Windows we could add that would test the positive case?